.mise.toml

```diff
@@ -0,0 +1,7 @@
+[tools]
+tuist = "4.23.0"
+swiftformat = "0.54.2"
```

Comment:
Nit: It'd be much easier to review this (and compare with other Mise migrations) if we stuck to the existing Swiftformat version and reverted the formatting changes.

Or if we are going to bump it let's go to the latest.